docs/intro/install.rst

```diff
@@ -38,6 +38,14 @@ By "official support", we mean that:
 -  Priority on bug reports is given to Apache and Nginx users if the bug is web server specific
    (but those are rare).
 
+.. note::
+
+   If using RHEL, CentOS, or any other distribution with SELinux enabled, you
+   will need to make sure that the appripriate contexts and permissions are configured:
```

Comment:
there is a typo here `appripriate ` needs to be `appropriate`

Reply:
Alas, spelling is hard. I fixed that, the commit message, and the extra file. For what it's worth, the documentation at http://learn.elgg.org/en/stable/contribute/docs.html#internationalizing-documentation implies (at least to me) that one *should* add the `.pot` file as part of the commit. If that's not right, maybe that area of documentation needs to be revised or clarified as to which situations do and do not need the translation files.